src/Tribe/RSVP.php

```diff
@@ -1314,7 +1314,7 @@ protected function get_attendees_by_order_id( $order_id ) {
 	 *
 	 * @return array
 	 */
-	protected function get_attendees( $attendees_query, $post_id ) {
+	protected function get_attendees( WP_Query $attendees_query, $post_id ) {
```

Comment:
Not necessarily a problem if we're bumping version requirements etc, but I wanted to flag this change to the method signature as other classes override this (so, potential compatibility concerns)